client/checkout/classic/upe.js

```diff
@@ -554,6 +577,7 @@ jQuery( function ( $ ) {
 					payment_method_data: {
 						billing_details: getBillingDetails( formFields ),
 					},
+					shipping: getShippingDetails( formFields ),
```

Comment:
I have to yet add similar check as for split-upe and include only for Afterpay. On first glance, I don't know how to get the used payment method type in this function

Reply:
`const paymentMethodType = getSelectedUPEGatewayPaymentMethod();`?

Reply:
@kalessil I'm getting weird `wc is not defined` error when using that method. By the look of it it seems the method works properly only for split UPE

Reply:
Oops, perhaps there is a similar one for legacy UPE?

Reply:
Found the way to do it in [this commit ](https://github.com/Automattic/woocommerce-payments/pull/6544/commits/0861a522415aa0906f37cd2d4476f4ba4dceddef)